js/templates/modals/orderDetail/summaryTab/orderDetails.html

```diff
@@ -110,10 +110,23 @@ <h2 class="tx4 margRTn"><%= ob.polyT('orderDetail.summaryTab.orderDetails.headin
         </div>
       </div>
       <hr class="clrBr" />
-      <div class="gutterVTn">
-        <div class="txB">Memo</div>
-        <div><%= item.memo ? ob.parseEmojis(item.memo) : ob.polyT('orderDetail.summaryTab.notApplicable') %></div>
+      <div class="flexRow gutterH">
+        <div class="col6">
+          <div class="gutterVTn">
+            <div class="txB">Memo</div>
+            <div><%= item.memo ? ob.parseEmojis(item.memo) : ob.polyT('orderDetail.summaryTab.notApplicable') %></div>
+          </div>
+        </div>
+        <div class="col6">
+          <div>
+            <div class="gutterVTn">
+              <div class="txB"><%= ob.polyT('orderDetail.summaryTab.orderDetails.alternateContact') %></div>
+              <div><%= ob.order.alternateContactInfo %></div>
```

Comment:
To follow suit with the other fields, can we show `ob.polyT('orderDetail.summaryTab.notApplicable')` if there is not alternate contact info set?

![image](https://user-images.githubusercontent.com/794517/30072628-ce3d9e88-921f-11e7-8293-4146b50cede1.png)

^^^ all empty fields show an n/a.